src/main/resources/elasticsearch.template.properties

```diff
@@ -0,0 +1,36 @@
+# ============ elasticsearch configuration for picturesafe-search =============
+#
+# picturesafe-search requires a configuration file named 'elasticsearch.properties' in the classpath,
+# in which the following default parameters can be overwritten.
+
+
+# --------------------------- Client configuration ----------------------------
+
+## List of host addresses to connect to (blank separated, format <hostname>:<port> or <ip>:<port>)
+#elasticsearch.hosts=localhost:9200
+
+## automatically discover nodes from a running Elasticsearch cluster and set them to an existing RestClient instance
```

Comment:
Set? Did you mean add?

Reply:
It's from the Elastic documentation, but I can change it...